modules/kafka-topic/inputs.tf

```diff
@@ -39,5 +39,10 @@ variable "max_message_bytes" {
 
 variable "consumer_prefix" {
   type        = string
-  description = "The prefix of the consumer group, by default"
+  description = "(Required) The prefix of the consumer group, by default"
+}
+
+variable "connector_service_account_id" {
+  type        = string
+  description = "(Required) The ID of the service account managing kafka connector"
```

Comment:
Minor: The `(Required)` is not really required, right? as it has no default.

Reply:
It required because It use to grant permission to the service account